composer.json

```diff
@@ -10,11 +10,9 @@
         }
     ],
     "require": {
-        "php":                      "^7.1",
-        "hostnet/asset-lib":        "dev-master",
-        "symfony/framework-bundle": "^3.3",
-        "symfony/http-foundation":  "^3.3",
-        "symfony/process":          "^3.3"
+        "php":               "^7.1",
+        "hostnet/asset-lib": "dev-master",
+        "symfony/symfony":   "^3.3"
```

Comment:
No `symfony/symfony` in open-source packages please, this will force people to get the whole framework rather than what they need